src/base/bittorrent/sessionimpl.cpp

```diff
@@ -3518,6 +3518,16 @@ void SessionImpl::setSaveResumeDataInterval(const int value)
     }
 }
 
+std::chrono::minutes SessionImpl::saveStatisticsInterval() const
+{
+    return std::chrono::minutes(m_saveStatisticsInterval);
+}
+
+void SessionImpl::setSaveStatisticsInterval(const std::chrono::minutes timeInMinutes)
```

Comment:
Am I the only one for whom it seems superfluous to mention twice that the value represents minutes?

Reply:
In a previous commit, the variable type was `int`. I've omitted to change its name after going for `std::chrono::minutes`